### ISSUE TITLE
Generate userflow based on Validation Workflow efforts

### ISSUE THREAD

**Issue by postphotos** [Automattic/amp-wp#1359]
As a developer learning about the AMP for WordPress plugin, I should have a clear userflow that describes how to trace and action on AMP errors on my site.

_Subissue breaking out the #1006 AC2 into a new task to keep the discussion focused here._

- [x] AC: As a developer, I should have clear instructions and a clear user diagram based on the new interfaces to understand how to action with AMP errors. These will be used in documentation and potentially in a future splash screen.

**Comment by kienstra** [comment]
**Question About Format Of User Flow**

Hi @postphotos,
Thanks for this issue.

>AC: As a developer, I should have clear instructions and a clear user diagram based on the new interfaces to understand how to action with AMP errors. These will be used in documentation and potentially in a future splash screen.

**Comment by jwold** [comment]
Based on a discussion today with @postphotos I believe we have a clear pathway to get to the error validation screens once you've activated the plugin. Would love any feedback:

<img width="1366" alt="validation user journey" src="https://user-images.githubusercontent.com/3411173/44753632-96d28300-aad3-11e8-8dcd-f03ddc4c132b.png">

**Comment by westonruter** [comment]
See also https://github.com/Automattic/amp-wp/pull/1183#issuecomment-416783852 for how there is now a pathway for a user to access the validation errors when they don't have native or paired modes enabled.

**Comment by postphotos** [comment]
@jwold @jillchu - Really like [this](https://github.com/Automattic/amp-wp/issues/1359#issuecomment-416755957). 

Can one of you generate one more userflow sketch of screens that explains what a user does past this? (How does a user get to each screen that @jillchu has generated?)

**Comment by postphotos** [comment]
> What format should this diagram have...a wiki page? And would just that satisfy this issue, or are there other formats?

I think this should probably be in the wiki, in other documentation and possibly incorporated into a future version of the splash screen. Essentially, I should know how errors are actually actioned. I think this is the simplest version of this:
1) I go to an error listing 
2) I select a URL or error (or isolate an error)
3) I action there
4) I can preview or view what changes happened. Repeat!

I can also:
1) Go to a URL while logged into my site
2) Click on the AMP status in the WP-admin
3) I action there
4) I can preview or view what changes happened. Repeat!

It's as easy as 1, 2, 3, we'd hope! 😸 

@kienstra @jwold

**Comment by jwold** [comment]
Following is the proposed user flow for installing the AMP plugin, choosing the mode you're site is in, and then working through validation errors as part of the compatibility tool. 

<img width="1366" alt="user flow" src="https://user-images.githubusercontent.com/3411173/45227971-a93c8180-b276-11e8-8407-9a45e3ffd6d6.png">

**Comment by jwold** [comment]
@amedina, @westonruter, @ThierryA 

We've made some changes to the diagram and updated it to a cleaner and higher fidelity wireframe. Would love any feedback, and also validation that things are all pointing to where they should (since I had to re-arrange some items).

Thank you!

![user flow](https://user-images.githubusercontent.com/3411173/45559015-355e2400-b7f6-11e8-9799-220ab9be9164.png)

cc @postphotos
